velox/docs/configs.rst

```diff
@@ -354,3 +354,15 @@ Spark-specific Configuration
      - bool
      - true
      - If false, ``size`` function returns null for null input.
+   * - spark.runtime.bloom_filter.expected_num_items
```

Comment:
Do we need .runtime in the property name? The other property, spark.legacy_size_of_null, doesn't have it.

Reply:
Yes, I think so. Because the corresponding spark configuration name is `spark.sql.optimizer.runtime.bloomFilter.expectedNumItems` https://spark.apache.org/docs/latest/configuration.html#runtime-sql-configuration

This config is the threshold to decide the bloom filter capacity at runtime.

Reply:
I see. Spark is a complete query engine with optimizer and runtime components. Velox is only the runtime, hence, no need to include 'runtime' in the configuration property names. Let's remove.